windows/micropython.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
> This, along with PlatformToolset, will make the project file unusable for earlier versions of VS (i.e. 2013).

Nevermind, seems I was wrong all along: opening the upgraded project in VS2013 just uses the v120 platformtoolset automatically so that worked out flawlessly after all..